### PR TITLE
Add babel polyfill and upgrade sanity base

### PR DESCRIPTION
**Pull request by devinhalladay**
This PR fixes an error when using the plugin with the latest beta update. It upgrades `@sanity/base` to a more recent stable version to resolve issues with the parts system, and adds `@babel/plugin-transform-runtime` to resolve this [error](https://stackoverflow.com/questions/33527653/babel-6-regeneratorruntime-is-not-defined) preventing the hosted package from building correctly.

I was able to test locally by creating another branch and pushing the `build` directory, then adding the git branch as an npm dependency in my sanity studio project.

Part of #8 but does not close it.

### CODE REVIEW

**Comment by devinhalladay** [comment]
Hi @KimPaow — I know you mentioned being busy these days, so I just wanted to bump this and make sure it is on your radar :) thank you again for your work on this library!

**Comment by KimPaow** [comment]
Thank you for this @devinhalladay! It's on my radar but still super busy, I might be able to take a look this weekend. Again, thanks for this contribution!

**Comment by devinhalladay** [comment]
Bumping this again as a reminder @KimPaow <3

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
     "description": "A custom input component for displaying an array of color to pick from.",
     "main": "build/index.js",
     "scripts": {
+        "prepare": "yarn build",
```
Comment (KimPaow) [inline package.json:7]: Thanks again for this and sorry for the delay, totally escaped my mind. Looks good, but could you run me through why this "prepare" script is needed?

**Review by KimPaow — COMMENTED** [review]
Nice work. Just need some clarification and I can have it out tomorrow.

**Comment by KimPaow** [comment]
@devinhalladay published 2.0.3. Thanks for the contribution!

**Pull request merged**